integration/firestore/gulpfile.js

```diff
@@ -17,6 +17,7 @@
 
 const del = require('del');
 const gulp = require('gulp');
+const process = require('process');
```

Comment:
Just curious about what this does?

Reply:
This brings in environment variables, which I am using to support both the memory and the persistence-based version of the integration test.

Reply:
Doesn't `process.env` just work without a require?

Reply:
It might - I was just trying to unblock my tests quickly. I will clean this up before I send this out for review.